__tests__/schemas.spec.ts

```diff
@@ -29,8 +29,8 @@ describe('Validate examples for cloud to device', () => {
         '$schemaName',
         // @ts-ignore
         ({ schemaName, schema, schemaTests }) => {
-            test.each<Schema>(schemaTests)('%o', example => {
-                expect(isValidSchema(schema, example)).toEqual(true);
+            test.each<Schema>(schemaTests)('%o', (example) => {
+                expect(isValidSchema(schema, example as any)).toEqual(true);
```

Comment:
The `example` parameter is type `Schema`. Please change it [here](https://github.com/nRFCloud/application-protocols/pull/29/files#diff-0750b726efeda903bc2f8d010fd965a777ae251ac437a7e96ebf843d9649e0f7R7) so you don't have to cast it as type `any`.